pkg/platform/aws/template.go

```diff
@@ -30,6 +30,7 @@ module "aws-{{.Config.ClusterName}}" {
   tags         = {{.Tags}}
   dns_zone     = "{{.Config.DNSZone}}"
   dns_zone_id  = "{{.Config.DNSZoneID}}"
+  set_csi_driver_iam_role = {{.Config.SetCSIDriverIAMRole}}
```

Comment:
Ah, CI fails, because this is not quoted. But if we change it to boolean, as I suggested, it should be fine then.